src/Fixer/Naming/HomoglyphNamesFixer.php

```diff
@@ -0,0 +1,250 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Naming;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Fred Cox <mcfedr@gmail.com>
+ */
+final class HomoglyphNamesFixer extends AbstractFixer
+{
+    /**
+     * Used the site https://github.com/mcfedr/homoglyph-download.
```

Comment:
1. why the fork? (https://github.com/mcfedr/homoglyph)
2. are the list of "types" (ipa, greek, cyrillic, ...) is just a full list of homoglyphs provided by that repo, or just a subset ?
3. why copy paste the code?
4. at https://github.com/mcfedr/homoglyph/blob/master/php/homoglyph.php#L6 there are 1560 homoglyphs, in your PR way less.
5. dataset of homoglyphs is not the code, should not be in code but in separated file storage